Game/Config/DefaultSpatialGDKSettings.ini

```diff
@@ -32,10 +32,8 @@ ActorGroups=()
 ServerWorkerTypes=("UnrealWorker")
 WorkerLogLevel=Warning
 bEnableUnrealLoadBalancer=False
```

Comment:
Maybe load balancing should be on by default in TestGyms now?

Reply:
I think it should be on by default now. That will make it much easier to test regressions.